### ISSUE TITLE
Erroring code example in AutoMM Presets page

### ISSUE THREAD

**Issue by Ludecan** [autogluon/autogluon#3351]
### Describe the issue linked to the documentation

Hey there. Just noticed an error in the code execution of the example of the Best Quality presets description here:
https://auto.gluon.ai/stable/tutorials/multimodal/advanced_topics/presets.html#best-quality

The code box is showing this:
```
╭─────────────────────────────── Traceback (most recent call last) ────────────────────────────────╮
│ in <module>:4                                                                                    │
│                                                                                                  │
│   1 from autogluon.multimodal import MultiModalPredictor                                         │
│   2                                                                                              │
│   3 predictor = MultiModalPredictor(label='label', eval_metric='acc', presets="best_quality"     │
│ ❱ 4 predictor.fit(train_data=train_data, time_limit=180)                                         │
│   5                                                                                              │
│                                                                                                  │
│ /home/ci/autogluon/multimodal/src/autogluon/multimodal/predictor.py:864 in fit                   │
│                                                                                                  │
│    861 │   │   │   )                                                                             │
│    862 │   │   │   return predictor                                                              │
│    863 │   │                                                                                     │
│ ❱  864 │   │   self._fit(**_fit_args)                                                            │
│    865 │   │   training_end = time.time()                                                        │
│    866 │   │   self._total_train_time = training_end - training_start                            │
│    867                                                                                           │
│                                                                                                  │
│ /home/ci/autogluon/multimodal/src/autogluon/multimodal/predictor.py:1140 in _fit                 │
│                                                                                                  │
│   1137 │   │   │   self._output_shape = len(df_preprocessor.label_generator.unique_entity_group  │
│   1138 │   │                                                                                     │
│   1139 │   │   if self._model is None:                                                           │
│ ❱ 1140 │   │   │   model = create_fusion_model(                                                  │
│   1141 │   │   │   │   config=config,                                                            │
│   1142 │   │   │   │   num_classes=self._output_shape,                                           │
│   1143 │   │   │   │   classes=self._classes,                                                    │
│                                                                                                  │
│ /home/ci/autogluon/multimodal/src/autogluon/multimodal/utils/model.py:442 in create_fusion_model │
│                                                                                                  │
│   439 │                                                                                          │
│   440 │   for model_name in names:                                                               │
│   441 │   │   model_config = getattr(config.model, model_name)                                   │
│ ❱ 442 │   │   model = create_model(                                                              │
│   443 │   │   │   model_name=model_name,                                                         │
│   444 │   │   │   model_config=model_config,                                                     │
│   445 │   │   │   num_classes=num_classes,                                                       │
│                                                                                                  │
│ /home/ci/autogluon/multimodal/src/autogluon/multimodal/utils/model.py:209 in create_model        │
│                                                                                                  │
│   206 │   │   │   pretrained=pretrained,                                                         │
│   207 │   │   )                                                                                  │
│   208 │   elif model_name.lower().startswith(HF_TEXT):                                           │
│ ❱ 209 │   │   model = HFAutoModelForTextPrediction(                                              │
│   210 │   │   │   prefix=model_name,                                                             │
│   211 │   │   │   checkpoint_name=model_config.checkpoint_name,                                  │
│   212 │   │   │   num_classes=num_classes,                                                       │
│                                                                                                  │
│ /home/ci/autogluon/multimodal/src/autogluon/multimodal/models/huggingface_text.py:84 in __init__ │
│                                                                                                  │
│    81 │   │   self.config, self.model = get_hf_config_and_model(                                 │
│    82 │   │   │   checkpoint_name=checkpoint_name, pretrained=pretrained, low_cpu_mem_usage=lo   │
│    83 │   │   )                                                                                  │
│ ❱  84 │   │   self._hf_model_input_names = AutoTokenizer.from_pretrained(checkpoint_name).mode   │
│    85 │   │                                                                                      │
│    86 │   │   if isinstance(self.model, T5PreTrainedModel):                                      │
│    87 │   │   │   self.is_t5 = True                                                              │
│                                                                                                  │
│ /home/ci/opt/venv/lib/python3.8/site-packages/transformers/models/auto/tokenization_auto.py:676  │
│ in from_pretrained                                                                               │
│                                                                                                  │
│   673 │   │   if model_type is not None:                                                         │
│   674 │   │   │   tokenizer_class_py, tokenizer_class_fast = TOKENIZER_MAPPING[type(config)]     │
│   675 │   │   │   if tokenizer_class_fast and (use_fast or tokenizer_class_py is None):          │
│ ❱ 676 │   │   │   │   return tokenizer_class_fast.from_pretrained(pretrained_model_name_or_pat   │
│   677 │   │   │   else:                                                                          │
│   678 │   │   │   │   if tokenizer_class_py is not None:                                         │
│   679 │   │   │   │   │   return tokenizer_class_py.from_pretrained(pretrained_model_name_or_p   │
│                                                                                                  │
│ /home/ci/opt/venv/lib/python3.8/site-packages/transformers/tokenization_utils_base.py:1804 in    │
│ from_pretrained                                                                                  │
│                                                                                                  │
│   1801 │   │   │   else:                                                                         │
│   1802 │   │   │   │   logger.info(f"loading file {file_path} from cache at {resolved_vocab_fil  │
│   1803 │   │                                                                                     │
│ ❱ 1804 │   │   return cls._from_pretrained(                                                      │
│   1805 │   │   │   resolved_vocab_files,                                                         │
│   1806 │   │   │   pretrained_model_name_or_path,                                                │
│   1807 │   │   │   init_configuration,                                                           │
│                                                                                                  │
│ /home/ci/opt/venv/lib/python3.8/site-packages/transformers/tokenization_utils_base.py:1959 in    │
│ _from_pretrained                                                                                 │
│                                                                                                  │
│   1956 │   │                                                                                     │
│   1957 │   │   # Instantiate tokenizer.                                                          │
│   1958 │   │   try:                                                                              │
│ ❱ 1959 │   │   │   tokenizer = cls(*init_inputs, **init_kwargs)                                  │
│   1960 │   │   except OSError:                                                                   │
│   1961 │   │   │   raise OSError(                                                                │
│   1962 │   │   │   │   "Unable to load vocabulary from file. "                                   │
│                                                                                                  │
│ /home/ci/opt/venv/lib/python3.8/site-packages/transformers/models/deberta_v2/tokenization_debert │
│ a_v2_fast.py:133 in __init__                                                                     │
│                                                                                                  │
│   130 │   │   mask_token="[MASK]",                                                               │
│   131 │   │   **kwargs                                                                           │
│   132 │   ) -> None:                                                                             │
│ ❱ 133 │   │   super().__init__(                                                                  │
│   134 │   │   │   vocab_file,                                                                    │
│   135 │   │   │   tokenizer_file=tokenizer_file,                                                 │
│   136 │   │   │   do_lower_case=do_lower_case,                                                   │
│                                                                                                  │
│ /home/ci/opt/venv/lib/python3.8/site-packages/transformers/tokenization_utils_fast.py:114 in     │
│ __init__                                                                                         │
│                                                                                                  │
│   111 │   │   │   fast_tokenizer = TokenizerFast.from_file(fast_tokenizer_file)                  │
│   112 │   │   elif slow_tokenizer is not None:                                                   │
│   113 │   │   │   # We need to convert a slow tokenizer to build the backend                     │
│ ❱ 114 │   │   │   fast_tokenizer = convert_slow_tokenizer(slow_tokenizer)                        │
│   115 │   │   elif self.slow_tokenizer_class is not None:                                        │
│   116 │   │   │   # We need to create and convert a slow tokenizer to build the backend          │
│   117 │   │   │   slow_tokenizer = self.slow_tokenizer_class(*args, **kwargs)                    │
│                                                                                                  │
│ /home/ci/opt/venv/lib/python3.8/site-packages/transformers/convert_slow_tokenizer.py:1162 in     │
│ convert_slow_tokenizer                                                                           │
│                                                                                                  │
│   1159 │                                                                                         │
│   1160 │   converter_class = SLOW_TO_FAST_CONVERTERS[tokenizer_class_name]                       │
│   1161 │                                                                                         │
│ ❱ 1162 │   return converter_class(transformer_tokenizer).converted()                             │
│   1163                                                                                           │
│                                                                                                  │
│ /home/ci/opt/venv/lib/python3.8/site-packages/transformers/convert_slow_tokenizer.py:438 in      │
│ __init__                                                                                         │
│                                                                                                  │
│    435 │   │                                                                                     │
│    436 │   │   super().__init__(*args)                                                           │
│    437 │   │                                                                                     │
│ ❱  438 │   │   from .utils import sentencepiece_model_pb2 as model_pb2                           │
│    439 │   │                                                                                     │
│    440 │   │   m = model_pb2.ModelProto()                                                        │
│    441 │   │   with open(self.original_tokenizer.vocab_file, "rb") as f:                         │
│                                                                                                  │
│ /home/ci/opt/venv/lib/python3.8/site-packages/transformers/utils/sentencepiece_model_pb2.py:92   │
│ in <module>                                                                                      │
│                                                                                                  │
│     89 │   file=DESCRIPTOR,                                                                      │
│     90 │   create_key=_descriptor._internal_create_key,                                          │
│     91 │   values=[                                                                              │
│ ❱   92 │   │   _descriptor.EnumValueDescriptor(                                                  │
│     93 │   │   │   name="UNIGRAM",                                                               │
│     94 │   │   │   index=0,                                                                      │
│     95 │   │   │   number=1,                                                                     │
│                                                                                                  │
│ /home/ci/opt/venv/lib/python3.8/site-packages/google/protobuf/descriptor.py:796 in __new__       │
│                                                                                                  │
│    793 │   def __new__(cls, name, index, number,                                                 │
│    794 │   │   │   │   type=None,  # pylint: disable=redefined-builtin                           │
│    795 │   │   │   │   options=None, serialized_options=None, create_key=None):                  │
│ ❱  796 │     _message.Message._CheckCalledFromGeneratedFile()                                    │
│    797 │     # There is no way we can build a complete EnumValueDescriptor with the              │
│    798 │     # given parameters (the name of the Enum is not known, for example).                │
│    799 │     # Fortunately generated files just pass it to the EnumDescriptor()                  │
╰──────────────────────────────────────────────────────────────────────────────────────────────────╯
TypeError: Descriptors cannot not be created directly.
If this call came from a _pb2.py file, your generated code is out of date and must be regenerated with protoc >= 
3.19.0.
If you cannot immediately regenerate your protos, some other possible workarounds are:
 1. Downgrade the protobuf package to 3.20.x or lower.
 2. Set PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python (but this will use pure-Python parsing and will be much 
slower).

More information: https://developers.google.com/protocol-buffers/docs/news/2022-05-06#
```

Keep up the good work. This library is awesome :muscle:

**Comment by jimmychordia** [comment]
Downgrading to protobuf package version 3.20.0 worked for me.

**Comment by zhiqiangdon** [comment]
Thanks for reporting this. We already fixed it in v0.8.1.
